src/Microsoft.DotNet.Interactive.PostgreSql.Tests/PSqlConnectionTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading.Tasks;
+using System.Linq;
+using System;
+using Microsoft.DotNet.Interactive.Tests.Utility;
+using Microsoft.DotNet.Interactive.Formatting.TabularData;
+using Microsoft.DotNet.Interactive.Formatting.Csv;
+using Microsoft.DotNet.Interactive.Formatting;
+using Microsoft.DotNet.Interactive.Events;
+using Microsoft.DotNet.Interactive.CSharp;
+using Microsoft.DotNet.Interactive.App;
+using FluentAssertions;
+
+namespace Microsoft.DotNet.Interactive.PostgreSql.Tests;
+
+public class PSqlConnectionTests : IDisposable
```

Comment:
I'd suggest renaming these classes with the `PSql` prefix to `PostgreSql*`. It makes things more consistent and easier to find.